### ISSUE TITLE
stagePadding on one side

### ISSUE THREAD

**Issue by ghost** [OwlCarousel2/OwlCarousel2#172]
Hi, is this possible?

**Comment by gaja33** [comment]
even im facing same problem im using stage padding i was able to remove padding left but i want to remove padding from right either and black space on owl-stage class..

**Comment by nuflowx** [comment]
When setting stagePadding it does so on the left on the right. Is it possible to only set it for the right side?

**Comment by nuflowx** [comment]
You can just set `padding-left: 0 !important` in your CSS file

**Comment by fcubria** [comment]
So anyone know if this is possible?
This is the case where this has sense: showing the user that there are more items without using prev and next buttons.
![stagepaddingjustoneside](https://cloud.githubusercontent.com/assets/3874641/24283985/1535a10c-1069-11e7-8ab6-8316b558bd82.jpg)
Thanks in advance.

**Comment by hellorich** [comment]
@fcubria This is exactly the implementation i'm trying to create - did you ever figure out a means of doing this?

**Comment by tungmv7** [comment]
Thanks @kevinmack18 Save my life!

**Comment by jhill-amaze** [comment]
@nuflowx I prefer your approach over @kevinmack18 because it avoids the need to keep two values in sync (css and js) instead of just one. Thanks!

**Comment by sushantvishwas1990** [comment]
@fcubria yes it is possible you have to use the **stagePadding**  with .owl-stage css property for example:

`<script type="text/javascript">
$('.your-selector').owlCarousel({
            items:1,
            stagePadding: 100,
            margin:10,
            loop:true,
            nav:false,
            dots:true,
        });
</script>`
<style>
    .owl-stage {
        left: -100px;
    }
</style>

**Comment by Sv9t** [comment]
@sushantvishwas1990 if I have 2 carousel in my page and I wont only 1 carousel left: -100px? What I do?

**Comment by Sv9t** [comment]
> @sushantvishwas1990 if I have 2 carousel in my page and I wont only 1 carousel left: -100px? What I do?

`<script type="text/javascript"> $('.your-selector1').owlCarousel({ items:1, stagePadding: 100, margin:10, loop:true, nav:false, dots:true, }); $('.your-selector2').owlCarousel({ items:1,margin:10, loop:true, nav:false, dots:true, });</script>`

<style>.your-selector1 .owl-stage { left: -100px; } </style>

**Comment by gopinath-twilightsoftwares** [comment]
On applying stagepadding with the left in negative works great. But, for the last item, it needs to be in the center while the loop: false. On loop: false, white space exists for the last item.

How can we fix this?

**Comment by Ejilarasan** [comment]
On applying stagepadding with the left in negative works great. But, for the last item, it needs to be in the center while the loop: false. On loop: false, white space exists for the last item.

How can we fix this?

**Comment by ydang204** [comment]
this can be fix by using 1.5 items but it can not swipe on mobile
any solution for this?

**Comment by ArunAnthony** [comment]
stepPdding leave's some space on after the items are over

**Comment by ArunAnthony** [comment]
StepPadding leaves blank space on right side when the number of items are completed after using right : -10 px important

**Comment by murtazajafari** [comment]
> I had the same issue, I resolved it by adding a negative `right:` of my `stagePadding`'s value on `.owl-stage` class .
> For example, `stagePadding: 20`:
> 
> ```css
> .owl-stage {
>     right: -10px;
> }
> ```

Greate, thank you. It worked for me, made my day

**Comment by Rahulv2492** [comment]
> I had the same issue, I resolved it by adding a negative `right:` of my `stagePadding`'s value on `.owl-stage` class .
> For example, `stagePadding: 20`:
> 
> ```css
> .owl-stage {
>     right: -10px;
> }
> ```

.owl-stage {
    padding-left: 0px !important;
}

padding-left to 0 fixed my issue
